docs/deployment/configuration.md

```diff
@@ -0,0 +1,93 @@
+# Configuration
+
+This section explain the current configuration options that are available for the different components that are deployed but the Helm charts.
+
+## Global
+
+The is only one mandatory configuration, which is the host name, for example:
+
+```yaml
+host: your-rdm-instance.com
+```
+
+Moreover, the services can be deployed along, note that it is recommended to deploy separatelly Elasticsearch and PostgreSQL for a production deployment.
+Therefore, by default only `redis` and `rabbitmq` are enabled. Example configuration:
+
+``` yaml
+postgresql:
+  inside_cluster: false
+
+elasticsearch:
+  inside_cluster: false
+```
+
+!!! info "inside_cluster availability
+    Note that the `inside_cluster` variable is supported for `redis`, `rabbitmq`, `elasticsearch`, `postgresql` and `haproxy`. The rest of the components
+    are mandatory.
+
+## HAProxy
+
+You can change the number of connections allowed by the haproxy with the `maxconn` variable:
+
+```
+haproxy:
+  maxconn: 100
+```
+
+!!! warning "Only one parent element"
+    By default the HAProxy is enabled, `inside_cluster` has the value `true`, nonetheless if you decide to set it, you only need
+    to specify once the `haproxy` parent in the yaml file. Otherwise the last one will override the previous. It should be something like:
+    ``` yaml
+    haproxy:
+        inside_cluster: true
+        maxconn: 100
+    ```
```

Comment:
I don't think I understand this part. Could you rephrase it?  :smiley_cat: (haproxy is enabled by default... so why do I need to specify it? What do I "set" and what is the "last one" that "will override the previous"?)

Reply:
It is a case that should not happen. If it is enabled you only, might, need to change the maxcoon. Otherwise just disable it. My point was more of a yaml syntax point. That it should be:

```yaml
haproxy:
    inside_cluster: true
    maxconn: 100
```

because if you do:

```yaml
haproxy:
    inside_cluster: true

haproxy:
    maxconn: 100
```

only the second one applies, but in this case doesn't really make sense since it is active by default. So I removed the paragraph.

Reply:
Ah got it.